src/snmp.c

```diff
@@ -99,6 +99,8 @@ struct host_definition_s {
   c_complain_t complaint;
   data_definition_t **data_list;
   int data_list_len;
+
```

Comment:
stray empty line

Reply:
Oh yes, i didn't know if they could have been packing issues with the struct so i added the field at the end but it didn't really "belongs" with thoses (sess_handle, complaint, data_list, data_list_len).

I don't really know the best place to put it in the struct either

Reply:
Oh yes, i didn't know if they could have been packing issues with the struct so i added the field at the end but it didn't really "belongs" with thoses (sess_handle, complaint, data_list, data_list_len).

I don't really know the best place to put it in the struct either

Reply:
At the end is fine